sorc/ncep_post.fd/GFSPOST.F

```diff
@@ -91,14 +113,20 @@ subroutine pvetc(km,p,px,py,t,tx,ty,h,u,v,av,hm,s,bvf2,pvn,theta,sigma,pvu)
 !> @param[in] h real (km) height (m).
 !> @param[in] t real (km) temperature (K).
 !> @param[in] pvu real (km) potential vorticity in PV units (10**-6*K*m**2/kg/s).
+!> @param[in] sigma real (km) static stability (K/m).
+!> @param[in] rh real (km) relative humidity.
+!> @param[in] omga real (km) vertical vorticity.
```

Comment:
vertical velocity in pressure coordinates?

Reply:
Ah, yes, I meant to put the units as a ? I just looked at the code and the units are 'Pa/s' which I suspected, but now confirmed.

Reply:
@hertneky Great! Added the Pa/s note for both.